src/NMEA/EngineState.hpp

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#pragma once
+
+#include "Validity.hpp"
+#include "time/Stamp.hpp"
+#include "Atmosphere/Temperature.hpp"
+
+/** Information about the engine sensor box values,
+ *  temperatures and revolutions per second
+*/
+struct EngineState
+{
+  Validity sensbox_flags_available;
+  uint8_t sensbox_flags;
```

Comment:
What is this? This sounds like it is very specific to your product, but it is undocumented. There should be documentation explaining that this is a proprietary attribute of your product, and there should be complete documentation on the meaning of the value.